backend/src/models/question.js

```diff
@@ -26,6 +15,19 @@ class Question {
             .query('SELECT * FROM question WHERE id=?', [questionID])
     }
 
+    /**
+     * @param   {string} code        The code of the couse we're getting questions for.
+     * @param   {number} pageNumber  The pageNumber, defaults to 1?, if higher than max should just give max.
+     * @returns {object}
+     */
+    getQuestions(code, pageNumber) {
+        let pageSize = 2
```

Comment:
Almost all vars should be `const` unless otherwise necessary.

Reply:
Noted and changed.